test/extensions_test.dart

```diff
@@ -887,6 +887,74 @@ void main() {
           expect(iterable(<num>[1, 3, 5, 9]).average, 4.5);
         });
       });
+      group('.min', () {
+        test('empty', () {
+          expect(() => iterable(<int>[]).min, throwsStateError);
+          expect(() => iterable(<double>[]).min, throwsStateError);
+          expect(() => iterable(<num>[]).min, throwsStateError);
```

Comment:
Where does `Iterable<num>.min` come from?

Reply:
IterableComparableExtension, updated that to also simplify the non-null cases to proxy to the nullable getter.

Reply:
Sweet!